src/Plugin/ServiceDefinition/EntityDelete.php

```diff
@@ -28,9 +29,16 @@ class EntityDelete extends ServiceDefinitionBase {
   /**
    * {@inheritdoc}
    */
+  public function processRoute(Route $route) {
+    $route->setRequirement('_entity_access', $this->getDerivativeId() . '.delete');
```

Comment:
I am curious what getDerivativeID is providing for us here? Is this some token or something?